docs/alba.js

```diff
@@ -0,0 +1,133 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+  const lam = 128;
+  const HASH_FACTOR = 1e8;
+
+  const node = document.getElementById('main_chart');
+  const n_p = document.getElementById('n_p');
+  const n_f = document.getElementById('n_f');
+  const item = document.getElementById('item');
+  const proof_size = document.getElementById('proof_size');
+  const s_p = document.getElementById('s_p');
+  const proof_proba = document.getElementById('proof_proba');
+
+  function U(n_p, n_f) {
+    return Math.ceil((lam + Math.log2(lam) + 5 - Math.log2(Math.log2(Math.E))) / Math.log2(n_p / n_f));
+  };
+
+  function probabilityOfProof(u, n_p, n_f, x) {
+    // from https://github.com/cardano-scaling/alba/discussions/17
+
+    S_1_bot = n_p / (17 ** 2 / (9 * Math.log2(Math.E)) * u ** 2) - 7 < 1;
+    S_2_bot = n_p / (17 ** 2 / (9 * Math.log2(Math.E)) * u ** 2) - 2 < 1;
+
+    if (!S_1_bot && !S_2_bot) {
+      throw Error("S_1_bot or S_2_bot");
+    }
```

Comment:
- Currently, this will just silently log this error to the console, potentially confusing users. Maybe use `alert` or sth like that?
 - One could also implement the remaining two cases in the proof of Corollary 3 in order to handle large `n_p` values (where this error is currently thrown).

(Doesn't necessarily have to happen in this PR)

Reply:
Yes, I copy/pasted your code. I think it's good enough for now but obviously throwing exceptions in user-facing code is not great. I don't like `alert()` but can find a good substitute.

Reply:
Why do you throw an error and not update the d/q/r parameters depending on S1 and S2?

Reply:
I think for this simulation we can just use the parameters for the "small case". They in fact work for all cases, it's just, the prover will not have a good worst-case running time. Then we don't need to worry about S_1_bot, etc.

We can borrow the math code from Raphael's implementation.
https://github.com/cardano-scaling/alba/pull/33/files#diff-b0bad697f839908964cc4e445cd9c87a76b4a5f84da347b24a426b61b7d09322R127